lib/main.dart

```diff
@@ -1,67 +1,305 @@
+    String gameShortDescription) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (gameName.length > 9) {
+    gameName = gameName.substring(0, 9) + "...";
+  }
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.8;
+  }
+  return Container(
+    margin:
+        const EdgeInsets.only(left: 10.0, right: 10.0, top: 10.0, bottom: 20.0),
+    child: Column(children: [
+      Container(
+        width: phoneSize.width * orientationWidth * 0.83 * 1.2 / 5,
+        height: phoneSize.width * orientationWidth * 0.83 * 1.2 / 5,
+        margin: const EdgeInsets.only(
+            left: 28.0, right: 28.0, top: 2.0, bottom: 10.0),
+        decoration: BoxDecoration(
+            border: Border.all(
+              color: Colors.indigo, //kHintColor, so this should be changed?
+            ),
+            borderRadius: const BorderRadius.all(Radius.circular(100)),
+            image: DecorationImage(image: AssetImage(link), fit: BoxFit.cover)),
+      ),
+      Row(children: [
+        Align(
+            alignment: Alignment.bottomCenter,
+            child: Text(
+              gameName,
+              style: const TextStyle(fontSize: 18, color: Color(0xFFf2f2f2)),
+            )),
+        const Icon(
+          Icons.verified,
+          size: 18,
+          color: Color(0xFFf2f2f2),
+        )
+      ]),
+    ]),
+  );
+}
+
+Widget _imageContainerThird(BuildContext context, String link) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.7;
+    orientationHeight = 2.1;
+    marginDivider = 2;
+  }
+  return Container(
+    width: phoneSize.width * 0.42 * orientationWidth,
+    height: phoneSize.height * 0.23 * orientationHeight,
+    margin: const EdgeInsets.all(16.0),
+    decoration: BoxDecoration(
+        borderRadius: const BorderRadius.all(Radius.circular(5.5)),
+        image: DecorationImage(image: AssetImage(link), fit: BoxFit.cover)),
+  );
+}
+
+Widget _textContainer(
+    BuildContext context, String mainText, String secondText) {
+  return Container(
+    margin: const EdgeInsets.all(12.0),
+    child: Row(
+      children: [
+        Text(
+          mainText,
+          style: const TextStyle(fontSize: 23, color: Color(0xFFf2f2f2)),
+        ),
+        Text(
+          secondText,
+          style: const TextStyle(fontSize: 12, color: Color(0xFFf2f2f2)),
+        )
+      ],
+    ),
+  );
+}
+
+Widget _iconBottom(BuildContext context, IconData icon, double height) {
+  return Icon(
+    icon,
+    size: height * 0.055,
+    color: Color(0xFFf2f2f2),
+  );
+}
+
+Widget _iconAppBar(BuildContext context, String icon, double width) {
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.4;
+
+  }
+  return Container(
+    margin: const EdgeInsets.only(
+      left: 10.0,
+      right: 10.0,
+    ),
```

Comment:
Might want to change it to
`EdgeInsets.symmetric(horizontal: 10.0)`
but this is fine as well :)